app/src/pages/inside/logsPage/defectEditor/itemHeader/itemHeader.jsx

```diff
@@ -76,11 +76,13 @@ ItemHeader.propTypes = {
   selectItem: PropTypes.func,
   nameLink: PropTypes.object,
   preselected: PropTypes.bool,
+  isShownLess: PropTypes.bool,
 };
 ItemHeader.defaultProps = {
   item: {},
   isSelected: false,
   selectItem: () => {},
   nameLink: {},
   preselected: false,
+  isShownLess: true,
```

Comment:
Please, pay attention this property apply to `copy from history line` section in modal window too. Seems like it should be `false` always for that section